shared/common-adapters/loading-wrapper.native.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import React, {Component} from 'react'
+import {Animated} from 'react-native'
+import Box from './box'
+import Button from './button'
+import HOCTimers from './hoc-timers'
+
+import type {TimerProps} from './hoc-timers'
+
+type Props = {
+  loading: boolean,
+  style?: Object,
+  doneLoadingComponent: React$Element<*>,
+  loadingComponent: React$Element<*>,
+  duration: number,
+  debugAnim?: boolean,
+} & TimerProps
+
+type State = {
+  opacity: any,
+  loadingActive: boolean,
+  loadingActiveTimeoutId: ?number,
+}
+
+class LoadingWrapper extends Component<void, Props, State> {
+  state: State;
+
+  constructor (props: Props) {
+    super(props)
+    this.state = {
+      opacity: new Animated.Value(1),
+      loadingActive: this.props.loading,
```

Comment:
Is it necessary to default this state value in this way? Looks like it gets set to true when the `loading` prop transitions values anyway?